File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-grouped-4.c
enum Status { DRAFT, REVIEW, REVISION, RETURNED, PUBLISHED, UNKNOWN };
enum Status status = DRAFT;

switch (status) {
    case DRAFT:
    case REVIEW:
    case REVISION:
    case RETURNED:
        open_editor();
        break;
    case PUBLISHED:
        display_article();
        break;
    default:
        show_status_hint();
        break;
}

// Continue after the selected action sequence.
continue_process();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/switch-case/code-examples/switch-grouped-4.cs
string status = "draft";

switch (status) {
    case "draft":
    case "review":
    case "revision":
    case "returned":
        OpenEditor();
        break;
    case "published":
        DisplayArticle();
        break;
    default:
        ShowStatusHint();
        break;
}

// Continue after the selected action sequence.
ContinueProcess();
